sklearn/utils/multiclass.py

```diff
@@ -432,17 +432,11 @@ def _ovr_decision_function(predictions, confidences, n_classes):
             votes[predictions[:, k] == 1, j] += 1
             k += 1
 
-    max_confidences = sum_of_confidences.max()
-    min_confidences = sum_of_confidences.min()
-
-    if max_confidences == min_confidences:
-        return votes
-
     # Scale the sum_of_confidences to (-0.5, 0.5) and add it with votes.
     # The motivation is to use confidence levels as a way to break ties in
     # the votes without switching any decision made based on a difference
     # of 1 vote.
     eps = np.finfo(sum_of_confidences.dtype).eps
-    max_abs_confidence = max(abs(max_confidences), abs(min_confidences))
-    scale = (0.5 - eps) / max_abs_confidence
-    return votes + sum_of_confidences * scale
+    sigmoid_confidences = ((np.exp(sum_of_confidences/10) - 1) /
```

Comment:
On what basis do you choose /10?

Reply:
not sure myself. I am worried about the behaviour of the sigmoid if the sum_of_confidences values are high.

Reply:
maybe a simpler function like
```python
lambda x: 0.5 - 1/(2*(x+1)) if x > 0 else -0.5 + 1/(2*(-x+1))
```
will be better, because we don't have to worry about overflow like with the exponential